lms/urls.py

```diff
@@ -147,6 +150,9 @@
 
     # robots.txt for mit oll
     url(r'^robots\.txt/$', TemplateView.as_view(template_name='robots.txt', content_type='text/plain')),
+
+    # redirecting due to high priority hotfix
+    url(r'^pre-biology', course_api_view.redirect_courses, name='redirect_single_course'),
```

Comment:
nit: just an alternate approach:
```suggestion
from django.views.generic.base import RedirectView

    url(r'^pre-biology', RedirectView.as_view(url='https://openlearninglibrary.mit.edu/courses/course-v1:OCW+Pre-7.01+1T2020/about'), name='redirect_single_course'),
```